src/js/common/schemaform/FormSaved.jsx

```diff
@@ -7,9 +7,19 @@ import Scroll from 'react-scroll';
 
 import { focusElement } from '../utils/helpers';
 import { fetchInProgressForm, removeInProgressForm } from './save-load-actions';
+import { handleVerify } from '../../common/helpers/login-helpers.js';
 
 import FormStartControls from './FormStartControls';
 
+moment.updateLocale('en', {
```

Comment:
@cehsu We should put this somewhere else, I think. It's going to affect all date formatting in form bundles, so it shouldn't be in here. Maybe FormApp? Not really sure.